code/modules/transport/pods/ships.dm

```diff
@@ -127,6 +127,23 @@
 
 ////////armed civ putt
 
+obj/machinery/vehicle/miniputt/pilot
+	New()
+		. = ..()
+		src.com_system.deactivate()
+		qdel(src.engine)
+		qdel(src.com_system)
+		src.components -= src.engine
+		src.components -= src.com_system
+		src.engine = new /obj/item/shipcomponent/engine/zero
```

Comment:
```suggestion
		src.engine = new /obj/item/shipcomponent/engine/zero(src)
```
You want the engine to exist in the pod.

Reply:
Done.